apps/server/src/modules/school/domain/type/file-storage-type.enum.ts

```diff
@@ -0,0 +1,3 @@
+export enum FileStorageType {
```

Comment:
Wollen wir hier ein Hinweis hinzufügen das FileStorageType und fileStorageKey in School Do nach dem entfernen der legacy files entfernt werden kann?

Reply:
Ja super.